docs/start.rst

```diff
@@ -46,7 +46,7 @@ The installation is quick and straightforward.
     airflow webserver --port 8080
 
     # start the scheduler
-    # open a new terminal or else run webserver with `-D` option to run it as a deamon
+    # open a new terminal or else run webserver with ``-D`` option to run it as a deamon
```

Comment:
Isn't this just a comment, that won't be rendered?

![image](https://user-images.githubusercontent.com/8811558/97348140-a7ced880-1885-11eb-8861-a2eb74ab91bc.png)